src/calculateRentalCost.js

```diff
@@ -5,6 +5,19 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  if (days === 7) {
+    return 230;
```

Comment:
The condition `days === 7` only accounts for exactly 7 days, but the discount should apply for 7 or more days. You need to adjust the condition to check for `days >= 7` instead.